test/stream-collateral/valid-default/expected-content/noStrict_default.js

```diff
@@ -0,0 +1,56 @@
+function Test262Error(message) {
+    if (message) this.message = message;
+}
+
+Test262Error.prototype.name = "Test262Error";
+
+Test262Error.prototype.toString = function () {
+    return "Test262Error: " + this.message;
+};
+
+function $ERROR(err) {
+  if(typeof err === "object" && err !== null && "name" in err) {
+      throw err;
+  } else {
+    throw new Test262Error(err);
+  }
+}
+
+function $DONE(err) {
+  if (err) {
+    if(typeof err === "object" && err !== null && "name" in err) {
+      print('test262/error ' + err.name + ': ' + err.message);
+    } else {
+      print('test262/error Test262Error: ' + err);
```

Comment:
Use template strings, unless there is an explicit value in using string concatenation. Also, I recognize that there is value in having test material that includes both.